user-service/src/main/java/by/battle/userservice/service/UserServiceImpl.java

```diff
@@ -0,0 +1,70 @@
+package by.battle.userservice.service;
+
+import by.battle.userservice.dto.UserDto;
+import by.battle.userservice.entity.Status;
+import by.battle.userservice.entity.User;
+import by.battle.userservice.exception.UserNotFoundException;
+import by.battle.userservice.mapper.UserDtoMapper;
+import by.battle.userservice.repository.RoleRepository;
+import by.battle.userservice.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class UserServiceImpl implements UserService {
+
+    private final UserRepository userRepository;
+    private final RoleRepository roleRepository;
+    private final UserDtoMapper userDtoMapper;
+
+    @Override
+    public List<UserDto> findAll() {
+        return userRepository.findAll().stream()
+                .map(userDtoMapper::mapToDto).collect(Collectors.toList());
+    }
+
+    @Override
+    public UserDto findById(UUID id) {
+        return userDtoMapper.mapToDto(userRepository.findById(id)
```

Comment:
let's do not be over paranoid for throwing UserNotFound.
NPE will be enough to understand wether user not found, unless you want to handle this somehow specifically.